blazor/blockeditor/globalization.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: Globalization in Blazor Block Editor Component | Syncfusion
+description: Checkout and learn about Globalization with Blazor Block Editor component in Blazor WebAssembly App.
+platform: Blazor
+control: Block Editor
+documentation: ug
+domainurl: ##DomainURL##
+---
+
+# Globalization in Blazor Block Editor Component
+
+The Syncfusion Blazor Block Editor component supports globalization features like `Localization` and `Right-to-Left (RTL)` alignment, enabling it to be adapted for different cultures and languages.
+
+## Localization
+
+Block Editor can be localized for any culture by translating its static text labels. The default locale is `en` (English). The following table lists the default text values for the `en` culture that can be translated.
+
+|KEY|Text|
```

Comment:
Can you check the other components? In blazor, we have resource file for localization